moshi/src/main/java/feign/moshi/MoshiDecoder.java

```diff
@@ -13,19 +13,16 @@
  */
 package feign.moshi;
 
-import com.google.common.io.CharStreams;
```

Comment:
Can you also remove the guava dependency from pom?

Reply:
Good point, done in https://github.com/OpenFeign/feign/pull/2183/commits/f3cd4d8e0b6d3575094d7af040879318bc417838